.github/workflows/stable-release-anaconda.yml

```diff
@@ -0,0 +1,34 @@
+name: Anaconda Stable Releases
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  conda-build-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          python-version: 3.8
+
+      - name: Install dependencies
+        shell: bash -l {0}
+        run: |
+          conda install -y pytorch torchvision cpuonly -c pytorch
+          pip install -r requirements-dev.txt
+          pip install --upgrade --no-cache-dir twine
```

Comment:
no need to install twine

Reply:
Are those 2 `pip install` steps required?

Reply:
Good question, need to check if we need to run `pip install -r requirements-dev.txt`

Reply:
@trsvchn can you check that please ?

Reply:
ok